src/main/resources/application.properties

```diff
@@ -24,6 +25,20 @@ cas.saml.tolerance=180000
 cas.send.renew=false
 
 # =========================================================================
+app.iam.request.form=https://www.hawaii.edu/bwiki/display/UHIAM/UH+Groupings+Request+Form
```

Comment:
As stated in the API PR, you can remove this line.

Reply:
done